src/hotspot/share/gc/g1/g1OopClosures.hpp

```diff
@@ -30,7 +30,7 @@
 #include "memory/iterator.hpp"
 #include "oops/markWord.hpp"
 
-class HeapRegion;
+class G1HeapRegion;
```

Comment:
With no other changes in this file, maybe this forward declaration isn't needed at all?  But keep for
now if removal leads to non-trivial additional changes.